server/chat-plugins/datasearch.ts

```diff
@@ -86,6 +79,7 @@ export const commands: ChatCommands = {
 			cmd: 'dexsearch',
 			canAll: (!this.broadcastMessage || (room && room.isPersonal)),
 			message: (this.broadcastMessage ? "" : message),
+		// @ts-ignore
```

Comment:
Why does this need to be ts-ignored now?

Reply:
`runSearch`'s return signature should still be defined in-file, right?

Reply:
because it has no designated type since not manually importing QueryProcessManager like i did before

Reply:
Shouldn't `QueryProcessManager` be a global? Otherwise how did it access it in the past?

Reply:
Ah, looks like it's imported inside this file. `QueryProcessManager` should still be imported at the top; it's just the other files that shouldn't.

Reply:
did